navigator/frontend/src/browsercheck.ts

```diff
@@ -37,6 +37,10 @@ const features = [
 
 // Find all features from above list that are not supported in current browser
 const missingFeatures = [];
+console.log(typeof Modernizr);
+console.log(Object.keys(Modernizr));
+console.log(Modernizr);
+console.log(Modernizr["es5undefined"]);
```

Comment:
I guess these are debug prints.

Reply:
yeah will clean it up once I figured out modernizr